### PR TITLE
Updates to Lack of Cache-Control for a Sensitive Page

### PR DESCRIPTION
**Pull request by RRudder**

### CODE REVIEW

File: submissions/description/server_security_misconfiguration/lack_of_security_headers/cache_control_for_a_sensitive_page/template.md [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+
+## Business Impact
+
+When sensitive data is being stored and transmitted by the application which does not have the `Cache-Control` header, an advanced attacker can access the sensitive data, phish users and cause reputational damage to the business.
+
+## Steps to Reproduce
+
+1. Enable a HTTP interception proxy, such as Burp Suite or OWASP ZAP
+1. Navigate to the following endpoint using a browser: {{URL}}
+1. Capture the request using the HTTP interception proxy
```
Comment (evildaemond) [inline submissions/description/server_security_misconfiguration/lack_of_security_headers/cache_control_for_a_sensitive_page/template.md:17]: Clarifying a technical note, the request does not show HTTP headers, it's the response. Updating this to suit will help both readers of the report and the customer. 

1. Capture the request using the HTTP interception proxy and review the response

File: submissions/description/server_security_misconfiguration/lack_of_security_headers/cache_control_for_a_sensitive_page/template.md [hunk 2]
```diff
@@ -0,0 +1,24 @@
...
+1. Observe that no `Cache-Control` header is implemented within the HTTP headers
```
Comment (evildaemond) [inline submissions/description/server_security_misconfiguration/lack_of_security_headers/cache_control_for_a_sensitive_page/template.md:18]: We need to have a form of impact here for a sensitive page, in this case, Cache Control on a sensitive page is usually shown by viewing the cache after the browser tab and session have been closed

**Review by evildaemond — CHANGES_REQUESTED** [review]
2 changes worth reviewing.

File: submissions/description/server_security_misconfiguration/lack_of_security_headers/cache_control_for_a_sensitive_page/template.md (hunk 2 above)
Comment (RRudder) [reply submissions/description/server_security_misconfiguration/lack_of_security_headers/cache_control_for_a_sensitive_page/template.md:18]: Cheers! @evildaemond GTG

**Review by evildaemond — APPROVED** [review]

**Pull request merged**